pkg/controller/vitesskeyspace/reconcile_resharding.go

```diff
@@ -120,8 +122,26 @@ func (r *reconcileHandler) reconcileResharding(ctx context.Context) (reconcile.R
 		sort.Strings(errorMsgs)
 		r.setConditionStatus(planetscalev2.VitessKeyspaceReshardingInSync, corev1.ConditionFalse, "Error", fmt.Sprintf("VReplication reported an error: %v", errorMsgs[0]))
 	case planetscalev2.WorkflowCopying:
+		// Aggregate tablet maps for all source shards.
+		sourceRowCount, err := r.shardsRowCount(ctx, workflowStatus.SourceShards)
+		if err != nil {
+			r.recorder.Eventf(r.vtk, corev1.EventTypeWarning, "SourceShardsRowCountFailed", "failed to aggregate row count for source shards: %v", err)
+			return resultBuilder.RequeueAfter(topoRequeueDelay)
+		}
```

Comment:
We should also check if `sourceRowCount == 0` so we don't divide by zero later. I guess in that case we can just return `99` since the copy is basically done and we're just waiting for the workflow to transition to Running.

Reply:
Good point

Reply:
Done.